code/modules/crafting/recipes.dm

```diff
@@ -173,8 +173,8 @@
 /datum/crafting_recipe/floorbot
 	name = "Floorbot"
 	result = /mob/living/simple_animal/bot/floorbot
-	reqs = list(/obj/item/storage/toolbox/mechanical = 1,
-				/obj/item/stack/tile/plasteel = 1,
+	reqs = list(/obj/item/storage/toolbox = 1,
+				/obj/item/stack/tile/plasteel = 10,
```

Comment:
intentional? lot of plasteel for a robot

Reply:
Those are ordinary tiles. Despite the name, they are not *actually* plasteel.

Reply:
Those are regular floortiles. The typepath just says plasteel because no one bothered to change it however long ago it was changed.

Reply:
oopsie whoopsie I knew that anyways not sure what I was thinking when I wrote that